invenio_rdm_records/records/systemfields/access/protection.py

```diff
@@ -8,48 +8,61 @@
 
 """Protection class for the access system field."""
 
+from enum import Enum
+
+
+class Visibility(Enum):
+    """Enum for the available visibility settings of a record's metadata and files."""
+
+    PUBLIC = "public"
+    RESTRICTED = "restricted"
+    PRIVATE = "restricted"
```

Comment:
No, it's just an alias for `restricted`, because it feels natural to me to have `private` be the opposite of `public`.
With auto-completion this shouldn't be too much of an issue though, and we can remove that alias if you prefer.